recipes/rocksdb/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "rocksdb" %}
+{% set version = "6.2.2" %}
+
+package:
+  name: rocksdb
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebook/rocksdb/archive/v{{ version }}.tar.gz
+  sha256: 3e7365cb2a35982e95e5e5dd0b3352dc78573193dafca02788572318c38483fb
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - cmake >=3.5.1,<4
```

Comment:
If you are using CMake, might as well use Ninja. IMO even with `-j${CPU_COUNT}`, make is sometimes slower compared to Ninja, plus Ninja doesn't require setting the -jX flag unless you want absolute control and want to use less memory/core.

Reply:
using ninja now...